src/bookshelf/README_SQL.md

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Should we nuke this before merging? I think that we've covered most of the TODOs in here that are critical.